packages/identity/CapacitorCommunityStripeIdentity.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'StripeIdentity', '23.4.0'
+  s.dependency 'StripeIdentity', '~> 23.4.0'
```

Comment:
Here is Identity's React Native plugin iOS version: https://github.com/stripe/stripe-react-native/blob/d565a0b1562b16ffcbf0932ef1ec4cbfb6b1e18f/stripe-react-native.podspec#L5